indico/modules/events/registration/controllers/display.py

```diff
@@ -490,6 +489,16 @@ def _process(self):
         return redirect(url)
 
 
+class RHTicketApplePass(RHTicketDownload):
```

Comment:
Let's add in the RH name the action (and perhaps follow the same name pattern for the Google Wallet one).

```suggestion
class RHApplePassTicketDownload(RHTicketDownload):
```

Reply:
To change it for Google Wallet i need @ThiefMaster approval.

Reply:
```suggestion
class RHTicketAppleWallet(RHTicketDownload):
```

Reply:
Done.
(I think that using RHTicketGoogleWalletDownload is formally wrong, because for Google when you click on that link you are not downloading a file but just browsing a new web page).